src/libsyncengine/update_detection/file_system_observer/computefsoperationworker.h

```diff
@@ -54,6 +54,8 @@ class ComputeFSOperationWorker : public ISyncWorker {
 
         void updateUnsyncedList();
 
+        void logOperationGeneration(const ReplicaSide side, const FSOpPtr fsOp);
```

Comment:
```suggestion
        void logOperationGeneration(ReplicaSide side, const FSOpPtr fsOp);
```

An argument passed by value has no side effect.

Reply:
Done